cmd/ro/main.go

```diff
@@ -380,12 +393,89 @@ func runResetPersisted() {
 	}
 
 	resp, err := roServer.ResetPersisted(req)
-	processError(err)
+	processError("error", err)
 
 	fmt.Println(resp.Status)
 	fmt.Println(resp)
 }
 
+func runSSHAgent() {
+	log.Println("Starting Red October Secret Shell Agent")
```

Comment:
I think SSH is Secure Shell.

Reply:
I know, I meant it to be a "Red October Secret Agent" joke but the "Shell" kinda ruins it :p